docs/textinput.md

```diff
@@ -437,7 +437,9 @@ Limits the maximum number of characters that can be entered. Use this instead of
 
 ### `multiline`
 
-If `true`, the text input can be multiple lines. The default value is `false`. It is important to note that this aligns the text to the top on iOS, and centers it on Android. Use with `textAlignVertical` set to `top` for the same behavior in both platforms.
+If `true`, the text input can be multiple lines. The default value is `false`. 
+
+> It is important to note that this aligns the text to the top on iOS, and centers it on Android. Use with `textAlignVertical` set to `top` for the same behavior in both platforms.
```

Comment:
This is the old notice formatting which will be replacing successfully with Docusuaurs admonitions. 

Please use `info` or `note` one instead  the blockquote in there:
* https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Update is done. hope I haven't screw anything up! 😄

Reply:
Looks good, however lint still complains about unnecessary space in line 440. Please check is there is no trailing spaces, or if the empty line aster the mentioned line did not contains space.